includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -545,4 +332,91 @@ private function getHref( $hrefKey ) {
 	private function getSocialHrefs() {
 		return DesignSystemSharedLinks::getInstance()->getSocialHrefs( $this->lang );
 	}
+
+	private function getLinkTextObjectList( String $section, Array $linkKeys ) {
```

Comment:
We'll probably extract those methods to separate classes, then it makes more sense to unittest it.